ldms/src/ldmsd/ldmsd.c

```diff
@@ -1571,6 +1571,20 @@ ldmsd_listen_t ldmsd_listen_new(char *xprt, char *port, char *host, char *auth)
 	return NULL;
 }
 
+static int __set_default_auth()
+{
+	ldmsd_auth_t auth_dom;
+	int rc = 0;
+	auth_dom = ldmsd_auth_new_with_auth(DEFAULT_AUTH, auth_name, auth_opt,
```

Comment:
Why isn't this just called ldmsd_auth_new? What does the extra _with_auth mean? I don't see an ldmsd_auth_new interface anywhere that would collide.

Also, in my checkout of this code, DEFAULT_AUTH == " _DEFAULT_AUTH_ ". This looks like a configuration issue of some kind, where we're not getting a reasonable default defined by configure.

Reply:
@narategithub: please respond to @tom95858 's questions.

Reply:
I went with `ldms_auth_new_with_auth()` to follow the other config objects signatures (e.g. prdcr_new_with_auth()). Our current use case does not have other credentials creating the auth object. So, I'm fine with refactoring this to just `ldms_auth_new()`.

The `DEFAULT_AUTH` is the key (name) of the default auth object. I use `" DEFAULT_AUTH "` so that it is guaranteed to not collide with user-specified names. The `-a` CLI option will set the `plugin` of the `DEFAULT_AUTH`, and the `-A` CLI options will set the auth options of the `DEFAULT_AUTH`.